integration-tests/cucumber-tests-platform-smartmetering/src/test/java/org/opensmartgridplatform/cucumber/platform/smartmetering/glue/steps/ws/smartmetering/smartmeteringconfiguration/UpdateFirmware.java

```diff
@@ -38,7 +38,8 @@ public class UpdateFirmware {
 
   @Autowired private SmartMeteringConfigurationClient client;
 
-  @Autowired private DeviceFirmwareModuleSteps deviceFirmwareModuleSteps;
+  @Autowired(required = false)
```

Comment:
I think `(required = false) ` can be removed

Reply:
Removed attribute _required_.